.github/workflows/release_community_bundle_parametric.yaml

```diff
@@ -0,0 +1,110 @@
+name: Community bundle
+on:
+  workflow_call:
+    secrets:
+      COMMUNITY_OPERATOR_TOKEN:
+        required: true
+    inputs:
+      version:
+        description: "The version to release, without the leading `v`"
+        required: true
+        type: string
+      previous_version:
+        description: "The previous version, used for the CVS's `replaces` field, without the leading `v`"
+        required: true
+        type: string
+      skip_range_lower:
+        description: "Lower bound for the skipRange field in the CSV, should be set to the oldest supported version, without the leading `v`"
+        required: true
+        type: string
+      community:
+        description: "The community to release the bundle to (either `k8s-operatorhub` or `redhat-openshift-ecosystem`)"
+        required: true
+        type: string
+      recipe:
+        description: "The recipe to use for creating the community bundle"
+        required: true
+        type: string
+
+permissions:
+  contents: write
+
+jobs:
+  make_community_bundle:
+    name: Build and commit PR for bundle in Operator Community
+    runs-on: ubuntu-22.04
+    env:
+      PROJECT: ${{ github.repository }}
+      VERSION: ${{ inputs.version }}
+      PREVIOUS_VERSION: ${{ inputs.previous_version }}
+      SKIP_RANGE_LOWER: ${{ inputs.skip_range_lower }}
+    steps:
+      - name: Log inputs
+        run: |
+          echo "Community: ${{ inputs.community }}"
+          echo "Recipe: ${{ inputs.recipe }}"
+          echo "Building version: ${VERSION}"
+          echo "which replaces version: ${PREVIOUS_VERSION}"
+          echo "Lower skip range bound: ${SKIP_RANGE_LOWER}"
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          path: operator
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: operator/go.mod
+
+      - name: Checkout Kubernetes Community Operators Hub
+        uses: actions/checkout@v3
+        if: ${{ inputs.community == 'k8s-operatorhub' }}
```

Comment:
Nice usage of conditional checkout based on inputs 👍🏼 
Maybe use more generic input values though, these look very coupled to the repo name, which would be a pity in case the repo name changes. Maybe just "k8s" and "OKD" is good enough?

Reply:
Thanks :)

If the repo name changes we still have to change this workflow, but not the callers, SMGT